bins/more/sim.fzb

```diff
@@ -158,109 +157,33 @@
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="36" moduleIdRef="a4f38a10-dd92-11dd-a3e2-001f5b3a17a4" path="h-bridge.fzp">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="36" moduleIdRef="1a7828b6466fac9fd042221e931124c4" path="74HC595.fzp">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="36" moduleIdRef="HEF4094" path="HEF4094.fzp">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance moduleIdRef="RevEd_FRM050">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance moduleIdRef="RevEd_ULN2003A">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="38" moduleIdRef="CrystalModuleID" path="crystal.fzp">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="38" moduleIdRef="37asdf3c843a2127a0e499f8a0b3ef6aCrystalModuleID" path="resonator-3pin.fzp">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="38" moduleIdRef="4001_4_x_2_input_NOR_gate_single">
+        <!--<instance moduleIdRef="AnodeLEDSuperfluxModuleID" modelIndex="24" path="led_flux_blue_anode.fzp"><views><iconView layer="icon"><geometry z="-1" x="-1" y="-1"/></iconView></views></instance><instance moduleIdRef="Prolight RGB 3 Watt Star" modelIndex="24" path="LED_Prolight_RGB.fzp"><views><iconView layer="icon"><geometry z="-1" x="-1" y="-1"/></iconView></views></instance><instance moduleIdRef="7segment10an" modelIndex="24" path="7segment_9,9_an.fzp"><views><iconView layer="icon"><geometry z="-1" x="-1" y="-1"/></iconView></views></instance><instance moduleIdRef="LED_Matrix_Kingbright_TA07-11" modelIndex="24" path="LED_Matrix_Kingbright_TA07-11.fzp"><views><iconView layer="icon"><geometry z="-1" x="-1" y="-1"/></iconView></views></instance>-->
```

Comment:
Can it be removed?

Reply:
Same as in the  previous comment: These lines are commented as these parts do not have SPICE fields currently, but they could be added more or less easily. It is a kind of TODO list for me, but we can remove them if you prefer it.